server/schedule/operator_controller.go

```diff
@@ -306,10 +318,10 @@ func (oc *OperatorController) addOperatorLocked(op *operator.Operator) bool {
 	// If there is an old operator, replace it. The priority should be checked
 	// already.
 	if old, ok := oc.operators[regionID]; ok {
+		_ = oc.removeOperatorLocked(old)
```

Comment:
Maybe we should handle this value?

Reply:
I don't think so, this function already get the lock, so we can be sure `oc.operators` won't change between line 320 and 321. The return value must be true.